pkg/utils/configs/config.go

```diff
@@ -407,3 +411,12 @@ connector {
 	qdrconfig.Execute(&buff, m.Spec)
 	return buff.String()
 }
+
```

Comment:
does the `router` section of the config need to be updated with saslConfig* or will it default to the correct location?

Reply:
It does and I had added it! I've made a mess when rebasing it would seem... sorry about that I'll fix up and push again.

Reply:
Sorry, I forgot I changed the mount to install it in the default location so it doesn't need to be specified in the router config.